pkg/translate/translate.go

```diff
@@ -330,92 +351,70 @@ func (t *Translator) ValuesOverlaysToHelmValues(in map[string]interface{}, cname
 	cur := out
 	for len(pv) > 1 {
 		cur[pv[0]] = make(map[string]interface{})
-		cur = cur[pv[0]].(map[string]interface{})
+		var ok bool
+		cur, ok = cur[pv[0]].(map[string]interface{})
+		if !ok {
```

Comment:
When would `cur[pv[0]]` not be of type `map[string]interface{}` given its always set on line 353?

Reply:
Done.